docs/web-apps/automated-testing/cypress/yaml/v1.md

```diff
@@ -897,6 +897,10 @@ cypress:
 In order for your additional reporter to work, it must be compatible with the [cypress-multi-reporter plugin](https://www.npmjs.com/package/cypress-multi-reporters), which provides the underlying functionality.
 :::
 
+:::warning
+This feature has been deprecated. We recommend configuring your reporters with your cypress config file. See the [cypress documentation](https://docs.cypress.io/guides/tooling/reporters) for details or our [example repo](https://github.com/saucelabs/saucectl-cypress-example/tree/main/v1/examples/multi-reporter) for an example implementation.
+:::
```

Comment:
Bit more assertive wording would be nice 🙏 Just to nudge them towards doing it, as it isn't optional.